snuba/cli/dlq_manager.py

```diff
@@ -0,0 +1,194 @@
+import os
+from typing import MutableSequence, Optional, Sequence, Tuple
+
+import click
+from arroyo import Message, Partition, Topic
+from arroyo.backends.kafka import KafkaConsumer, KafkaPayload
+
+from snuba.datasets.storages.factory import WRITABLE_STORAGES
+from snuba.utils.streams.configuration_builder import build_kafka_consumer_configuration
+
+__STORAGE_TO_DLQ_TOPIC = {
+    storage: storage.get_table_writer()
+    .get_stream_loader()
+    .get_dead_letter_queue_topic()
+    for storage in WRITABLE_STORAGES.values()
+}
+
+STORAGE_SETS_WITH_DLQ = {
+    storage.get_storage_set_key().value: topic
+    for storage, topic in __STORAGE_TO_DLQ_TOPIC.items()
+    if topic is not None
+}
+
+LINE_BREAK = "-" * 50
+
+
+@click.group()
+def dlq_manager() -> None:
+    """
+    Tools for Snuba's Dead Letter Queues
+    """
+    pass
+
+
+@dlq_manager.command()
+@click.option(
+    "--storage-set",
+    "storage_set",
+    type=click.Choice([storage_set for storage_set in STORAGE_SETS_WITH_DLQ]),
+    help="The storage set to list dead letters for",
+    required=True,
+)
+@click.option(
+    "--offset",
+    "offset",
+    type=int,
+    help="An offset on the dead-letter topic to start reading from, earliest by default",
+    default=0,
```

Comment:
The description says the earliest is the default but then it's set to 0 specifically, which may not be the earliest one. What happens if 0 is out of range?